tests/ui/storage_class/push_constant.rs

```diff
@@ -7,6 +7,7 @@
 use spirv_std as _;
 
 #[derive(Copy, Clone)]
+#[spirv(block)]
```

Comment:
This test has a comment about how it should *only* pass as `spirv-unknown-unknown`, effectively.

I wonder if you can filter the target with a compiletest directive, or maybe add `#[cfg(target_env = "unknown")]` on the `struct` and the `fn`?

Reply:
Also there's a separate old test that you can now move into `tests/ui`, which tests the Vulkan validation side of this.